plotting/plotter_vh_trigTest.py

```diff
@@ -0,0 +1,1121 @@
+          continue 
+        else:
+          newsamples[s] = samples[s]
+      else:
+        newsamples[s] = samples[s]
+    samples = newsamples
+
+  plots   = plotsFile.plots
+  if options.sample != []:
+    newsamples = {}
+    for s in options.sample:
+      for ss in samples.keys():
+        if re.match(s, ss):
+          print("Will run over:", ss)
+          newsamples[ss] = samples[ss]
+    samples = newsamples
+  if options.exclude != []:
+    newsamples = {}
+    for ss in samples.keys():
+      add = True
+      for s in options.exclude:
+        if re.match(s, ss):
+          print("Will exclude sample:", ss)
+          add = False
+      if add:
+        newsamples[ss] = samples[ss]
+    samples = newsamples
+
+  if options.test:
+    for s in samples:
+      samples[s]["files"] = [samples[s]["files"][0]]
+
+  if options.blind:
+    toDelete = []
+    for s in samples:
+      if "data" in s: 
+        toDelete.append(s)
+    for d in toDelete:
+      del samples[d]
+  if len(options.singleplot) > 0:
+    newplots = {p: plots[p] for p in options.singleplot}
+    plots = newplots
+
+  if options.files:
+    for s in samples:
+      newfiles = []
+      for f in samples[s]["files"]:
+        if f in options.files:
+          #print(f)
+          newfiles.append(f)
+      samples[s]["files"] = newfiles
+
+  thePlotter = plotter(plots, samples, options)
+  if options.queue:
+    print(options.jobname, os.path.isdir(options.jobname))
+    if not(os.path.isdir(options.jobname)):
+      print("mkdir %s"%options.jobname) 
+      os.system("mkdir %s"%options.jobname)
+    if not(os.path.isdir(options.jobname+"/exec")):
+      os.system("mkdir %s/exec"%options.jobname)
+    else:
+      os.system("rm %s/exec/*"%options.jobname)
+    if not(os.path.isdir(options.jobname+"/batchlogs")):
+      os.system("mkdir %s/batchlogs"%options.jobname)
+
+    command = "python plotter_vh.py " + subprocess.list2cmdline(sys.argv[1:]).replace("--queue %s"%options.queue,"")
+    savecommand = open("%s/command.txt"%options.toSave,"w")
+    savecommand.write(command+ "\n")
+    savecommand.close()
+    nJobs = thePlotter.createJobs(options, command)
+    os.system("rm submit.sub")
+    subfile = open("submit.sub", "w")
+    subfile.write("executable              = $(filename)\n")
+    subfile.write("arguments               = $(ClusterId)$(ProcId)\n")
+    subfile.write("output                  = %s/$(ClusterId).$(ProcId).out\n"%("%s/batchlogs"%options.jobname))
+    subfile.write("error                   = %s/$(ClusterId).$(ProcId).err\n"%("%s/batchlogs"%options.jobname))
+    subfile.write("log                     = %s/$(ClusterId).log\n"%("%s/batchlogs"%options.jobname))
+    subfile.write("+JobFlavour = \"%s\"\n"%(options.queue))
+    subfile.write("queue filename matching (%s/exec/_*sh)\n"%("%s"%options.jobname))
+    subfile.close()
+    if not(options.pretend): os.system("condor_submit -spool submit.sub")
+
+  else:
+    thePlotter.doPlots(options)
```

Comment:
Try not to commit carbon copies of other files like this, if you did something to debug locally either delete it at the end or -if changing features- modify the central file. Makes no sense to have 20 copies of the same thing lying around